doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -798,6 +798,7 @@ Modem libraries
   * Deprecated the Kconfig option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_ZEPHYR`.
   * Fixed an issue with the CFUN hooks when the Modem library is initialized during ``SYS_INIT`` at kernel level and makes calls to the :ref:`nrf_modem_at` interface before the application level initialization is done.
   * Removed the deprecated options ``CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_ASYNC`` and ``CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_SYNC``.
+  * Fixed a potential issue where the Modem library would not wait for the Modem to complete the coredump before shutting it down after a modem fault.
```

Comment:
Nit: we have some inconsistencies that I noticed also on other documents. We always write `Modem library`, which is ok, but when we write `modem` I don't think we should capitalize the `m`.

Either (preferred):

```suggestion
  * Fixed a potential issue where the Modem library would not wait for the modem to complete the coredump before shutting it down after a modem fault.
```

Or (consistency):

```suggestion
  * Fixed a potential issue where the Modem library would not wait for the Modem to complete the coredump before shutting it down after a Modem fault.
```